backtrace-library/src/main/java/backtraceio/library/models/BacktraceData.java

```diff
@@ -98,103 +97,198 @@ public class BacktraceData {
     /**
      * Current BacktraceReport
      */
-    public transient BacktraceReport report;
-
-    /**
-     * Current application context
-     */
-    public transient Context context;
+    public transient BacktraceReport report; // Think if we need it
 
     /**
      * Application thread details
      */
     @SerializedName("threads")
     Map<String, ThreadInformation> threadInformationMap;
+    
+    public BacktraceData() { }
```

Comment:
we didn't need empty constructor in the past. Do we need it again?

Reply:
We will need it in next PR but in this PR I will remove it, done.